docs/governance/working-groups/security/charter.md

```diff
@@ -0,0 +1,52 @@
+# Security Working Group
+
+This [WG] defines how new and existing security controls catalogs can be used to assess the security of the common cloud services defined by the CCC services taxonomy working group.
+
+## Mission
+
+The mission of the Security [WG] is to develop, maintain, and enhance a comprehensive threat-informed security controls catalog specifically tailored to the CCC cloud services taxonomy. Additionally, the working group will explore how existing security controls catalogs can be leveraged to assess the security posture of cloud services, define methodologies for conducting such assessments, and ensure alignment with industry standards and best practices.
+
+## Approach
+
+- Maintain open and transparent communication channels for all WG activities and decisions in regard to outputs/deliverables
+- Maintain a list of all community outputs that need to be released
+- Determine best process for versioning/releasing the output owners content
+
+## Responsibilities
+
+- Create and maintain a detailed security controls catalog outlining effective measures to mitigate identified threats.
```

Comment:
For completeness, are we generating a single catalog of controls and threat information embedded as opposed to a seperate catalog for controls and a seperate catalog for threats that references one another?

Reply:
@ rowan-baker - we should keep the threats and control catalogs separate, and associate the IDs for each threat to each control. A threat can have 1 or more controls associated with it. With that being stated, having separate catalogs makes it easier to manage, read, and scale. It'll also make rebasing a lot easier (in the event we run into merging problems/issues).

Reply:
Agree with @damienjburks  - and our examples so far are based on having a separate catalog for threats and a separate catalog for controls. In my mind, what matters most for CC is a _threat-informed_ **controls catalog** and so that gets the explicit mention in the WG charter. Although a separate example threat catalog is currently helping elucidate some of our early brainstorming, how much CCC wants to invest in a separate unique threat catalog is still a little unclear.